docs/dev/clone_ferc1.rst

```diff
@@ -35,30 +35,29 @@ database cloning and connect directly to the resulting database. This has become
 especially useful since Microsoft recently discontinued the database driver that until
 late 2018 had allowed users to load the FoxPro database files into Microsoft Access.
 
-In any case, cloning the original FERC database is the first step in the PUDL
-ETL process. This can be done with the ``ferc_to_sqlite`` script (which is an
-entrypoint into the :mod:`pudl.convert.ferc_to_sqlite` module) which is
-installed as part of the PUDL Python package. It takes its instructions from a
-YAML file, an example of which is included in the ``settings`` directory in
-your PUDL workspace. Once you've :ref:`created a datastore <datastore>`, you can
-try this example:
+Cloning the original FERC database is the first step in the PUDL
+ETL process. This can be done using the dagster UI (see :ref:`run-dagit`)
+or with the ``ferc_to_sqlite`` script (see :ref:`run-cli`).
 
-.. code-block:: console
+.. note::
 
-   $ ferc_to_sqlite settings/etl_full.yml
+  We recommend using Dagit to execute the ETL as it provides additional
+  functionality for re-execution and viewing dependences.
 
-This should create several outputs that you can find in your workspace at:
+Executing a ``ferc_to_sqlite`` job will create several outputs that you can
+find in your workspace at:
 
- * ``sqlite/ferc1.sqlite``: Data from 1994-2020 (FoxPro/DBF)
- * ``sqlite/ferc1_xbrl.sqlite``: Data from 2021 onward (XBRL)
- * ``sqlite/ferc1_xbrl_datapackage.json``: `Frictionless data package
+ * ``outputs/ferc1.sqlite``: Data from 1994-2020 (FoxPro/DBF)
+ * ``outputs/ferc1_xbrl.sqlite``: Data from 2021 onward (XBRL)
+ * ``outputs/ferc1_xbrl_datapackage.json``: `Frictionless data package
    <https://specs.frictionlessdata.io/data-package/>`__ descriptor for the XBRL derived
    database.
- * ``sqlite/ferc1_xbrl_taxonomy_metadata.json``: A JSON version of the XBRL Taxonomy,
+ * ``outputs/ferc1_xbrl_taxonomy_metadata.json``: A JSON version of the XBRL Taxonomy,
```

Comment:
If we settled on the singular `PUDL_OUTPUT` and "output" as the default in the warnings and what gets created if someone is migrating from the old setup these paths should match that too.